plonetheme/onegovbear/browser/forms.py

```diff
@@ -0,0 +1,148 @@
+from persistent.mapping import PersistentMapping
+from plone.directives import form
+from plone.z3cform.layout import wrap_form
+from plonetheme.onegovbear import _
+from plonetheme.onegovbear.browser.fields import Variable
+from zope.annotation import IAnnotations
+from zope.interface import implements
+
+ANNOTATION_KEY = 'plonetheme.onegovebear.custom_scss_variables'
+
+
+class IVariablesSchema(form.Schema):
+    primary_color = Variable(
+        variable_name='$primary-color')
```

Comment:
By removing `variable_name=` each variable definition should fit on one line, right?
This would make the list better readable.
Just an idea :wink: